src/request.rs

```diff
@@ -9,6 +10,12 @@ use async_std::task::{Context, Poll};
 use std::pin::Pin;
 use std::{str::FromStr, sync::Arc};
 
+use crate::error::{Error, ResultExt, StringError};
+use crate::middleware::cookies::CookieData;
+
+const MIDDLEWARE_MISSING_MSG: &str =
+    "CookiesMiddleware must be used to populate request and response cookies";
```

Comment:
Though as mentioned in https://github.com/http-rs/tide/pull/380#issuecomment-576795696 we might not need a feature flag at all if we can make the parsing occur on first access of the cookies: https://github.com/http-rs/tide/pull/380#issuecomment-576795696.

Unsure how well this would work with https://github.com/http-rs/tide/pull/380#discussion_r369169063 though.